types/Batch/BatchStatuses.d.ts

```diff
@@ -0,0 +1,3 @@
+import { TBatchStatus } from './BatchStatus';
+
+export declare type TBatchStatuses = { [key in TBatchStatus]: number };
```

Comment:
What does the `number` do here? It seems like this is making a list of all the strings in `TBatchStatus`?

Reply:
Maybe this helps to show what it's doing. For each key in the list of statuses, it will have a value of how many shipments are in each status for the batch:

![Screen Shot 2022-07-05 at 11 32 05 AM](https://user-images.githubusercontent.com/39606064/177383908-2f38ed5e-5db8-4182-adcf-1aec75b110bc.png)